chart/cas-cif/values.yaml

```diff
@@ -39,7 +39,7 @@ hostName: ~
 
 certbot:
   image:
-    tag: "1.0"
+    pullPolicy: IfNotPresent
```

Comment:
Any reason to override this value from the original chart?

Reply:
I see the default pullPolicy is 'Always' in the original chart. Which I figured was unnecessary